searx/settings.yml

```diff
@@ -1570,7 +1570,7 @@ engines:
   - name: internetarchivescholar
     engine: internet_archive_scholar
     shortcut: ias
-    timeout: 5.0
+    timeout: 15.0
 
```

Comment:
On common science term I have a response time of 2-3 sec .. the word `test` is in nearly all papers .. (I have response time 6-8sec) .. I would suggest to increase the response time to 8 and to use a search term other than `test` in the checker.

Reply:
Why not 15 ?

Reply:
Why not 1h? .. since it is to long, when we have at maximum a response time of 8sec?

Did you had 15sec response time in your sample tests .. or why did you think 15sec is the best value?

Reply:
I have a 11 seconds response time for `time`.

Reply:
> I have a 11 seconds response time for `time`.

Hm .. haven't thought a request took longer than 8sec .. but you are right: `time` has a response time > 11sec